jdbc/src/main/java/com/qubole/quark/jdbc/QuarkDriver.java

```diff
@@ -101,7 +101,10 @@ public Meta createMeta(AvaticaConnection connection) {
         ((QuarkConnectionImpl) connection).getProperties());
   }
 
-  public String getSchemaFactoryPath() {
+  public String getSchemaFactoryPath(boolean isDbSchema) {
```

Comment:
This needs to be cleaned up. I suggest accepting the schemafactory in the JSON and default to JSON schemafactory